### ISSUE TITLE
Email: Page Title missing for HTML documents.

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#611]
## Notes
The permit-rules.html document attached to the email is missing a page title. 

![image.png](https://images.zenhubusercontent.com/5913513251aa2c30577de73e/6b74cbe3-090e-44ea-8d3a-08701b1b0a15)

Linkage: CTF-94
WCAG: 2.4.2 Page Titled 

## Acceptance criteria
- [x] permit rules html has title

## Definition of Done
- [x] Code unit / e2e tested and comply with QWASP
- [x] No new security vulnerabilities
- [x] Internal team code review
- [x] Documentation / readme.md / CircleCI updated
- [x] Docker updated
- [x] Compare finished design with mockup
- [ ] Usability tested

**Comment by sadlerw** [comment]
<img width="754" alt="screen shot 2018-03-22 at 2 24 01 pm" src="https://user-images.githubusercontent.com/2058839/37793534-fe62f28a-2ddc-11e8-9491-4ba39cb6259d.png">
